src/lib/y2network/sysconfig/connection_config_readers/bonding.rb

```diff
@@ -37,7 +37,7 @@ def update_connection_config(conn)
         #
         # @return [Array<String>] bonding slaves defined in the file
         def slaves
-          (file.bonding_slaves || {}).values
+          (file.bonding_slaves || {}).sort.to_h.values
```

Comment:
that sort do it via keys? and alphabetical? This can be problem if there are more then 10 bonds, but I expect it cannot happen

Reply:
Yep, I was already thinking on that, but If I'm not wrong we limited it to 8 at least in the past. But we could convert  it to integer to be safe.